content/pt-br/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -18,122 +15,93 @@ princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#ku
 
 ```shell
-kubectl get SampleDB                   # encontra a base de dados configurada
+kubectl get SampleDB                   # encontrar banco de dados configurados
 
-kubectl edit SampleDB/example-database # mudar manualmente algumas definições
+kubectl edit SampleDB/example-database # alterar manualmente algumas configurações
 ```
 
-&hellip;e é isso! O Operador vai tomar conta de aplicar
-as mudanças assim como manter o serviço existente em boa forma.
+&hellip;e é isso! O Operador cuidará de aplicar as alterações, bem como manter o serviço existente em bom estado.
+
+## Escrevendo o seu próprio operador
 
-## Escrevendo o seu próprio Operador {#escrevendo-operador}
+Se não houver um opeardor no ecosistema que implemente o comportamento Desejado, você pode programar o seu próprio.
 
-Se não existir no ecosistema um Operador que implementa
-o comportamento que pretende, pode codificar o seu próprio.
-[Qual é o próximo](#qual-é-o-próximo) você vai encontrar
-alguns *links* para bibliotecas e ferramentas que pode usar
-para escrever o seu próprio Operador *cloud native*.
+Você também pode implementar um operador (ou seja, um Controlador) usando qualquer linguagem/agente de execução que possa atuar como um [cliente para a API do Kubernetes](/docs/reference/using-api/client-libraries/).
 
-Pode também implementar um Operador (isto é, um Controlador) usando qualquer linguagem / *runtime*
-que pode atuar como um [cliente da API do Kubernetes](/docs/reference/using-api/client-libraries/).
+A seguir estão algumas bibliotecas e ferramentas que você pode usar para escrever seu próprio operador nativo de nuvem.
 
+> **_NOTE:_**  Esta seção contém links para projetos de terceiros que fornecem funcionalidades requeridas pelo Kubernetes. Os autores do projeto Kubernetes não são responsáveis por esses projetos, que estão listados em ordem alfabética. Para adicionar um projeto a esta lista, leia o [guia de conteúdo](/docs/contribute/style/content-guide/#third-party-content) antes de enviar uma alteração. [Mais informações](/docs/concepts/extend-kubernetes/operator/#third-party-content-disclaimer).
 
+{{% thirdparty-content %}}
 
-## {{% heading "whatsnext" %}}
+* [Charmed Operator Framework](https://juju.is/)
+* [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk)
+* [Kopf](https://github.com/nolar/kopf) (Kubernetes Operator Pythonic Framework)
+* [kube-rs](https://kube.rs/) (Rust)
+* [kubebuilder](https://book.kubebuilder.io/)
+* [KubeOps](https://buehler.github.io/dotnet-operator-sdk/) (.NET operator SDK)
+* [KUDO](https://kudo.dev/) (Kubernetes Universal Declarative Operator)
+* [Mast](https://docs.ansi.services/mast/user_guide/operator/)
+* [Metacontroller](https://metacontroller.github.io/metacontroller/intro.html) juntamente com *WebHooks* que você mesmo implementa
+* [Operator Framework](https://operatorframework.io)
+* [shell-operator](https://github.com/flant/shell-operator)
 
+## O que vem a seguir
```

Comment:
This should be reverted back to the `{{% heading "whatsnext" %}}` shortcode. It gets translated automatically.